### PR TITLE
Update ja.json & Swipe gestures indicators

### PR DESCRIPTION
**Pull request by LEoREo2247**
I just filled missing translation and fixed some very small translation error and added horizontal swipe gestures indicators like chrome

### CODE REVIEW

File: css/webviews.css [hunk 1]
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
 	height: 100vh;
 	z-index: 99;
 }
+
+.arrow-indicator {
+	width: 140px;
+	height: 140px;
+	border-radius: 70px;
+	background: rgba(0,0,0,0.8);
```
Comment (PalmerAL) [inline css/webviews.css]: Since there's another rule below to set the opacity, can this just be ```#000```?

File: css/webviews.css [hunk 2]
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
...
+	position: fixed;
+	top: 50%;
+	z-index: 1;
+	transition: opacity 0.3s, visibility 0.3s;
```
Comment (PalmerAL) [inline css/webviews.css]: There's no transition for ```visibility```, so I think that part can be removed without affecting anything.

File: css/webviews.css [hunk 3]
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
...
+}
+#left {
+	left: -70px;
+}
+#right {
+	right: -70px;
+}
+
+.arrow {
+	color: white;
```
Comment (PalmerAL) [inline css/webviews.css]: The indentation here doesn't match the other lines

File: index.html [hunk 4]
```diff
@@ -61,7 +61,14 @@
 
 	<div id="overlay"></div>
 
-	<div id="webviews"></div>
+	<div id="webviews">
+		<div id="left" class="arrow-indicator">
```
Comment (PalmerAL) [inline index.html]: Can you make the id's of these something more descriptive, like "leftArrowContainer"?

File: css/webviews.css [hunk 5]
```diff
@@ -32,3 +32,40 @@ body.windows #webviews {
...
+	background: rgba(0,0,0,0.8);
+	position: fixed;
+	top: 50%;
+	z-index: 1;
+	transition: opacity 0.3s, visibility 0.3s;
+	opacity: 0;
+	visibility: hidden;
+}
+.arrow-indicator.shown {  
+	opacity: 1;
```
Comment (PalmerAL) [inline css/webviews.css]: I think it might be better to have this less opaque - maybe around 0.6?

File: js/webviews.js [hunk 6]
```diff
@@ -274,6 +274,22 @@ webviews.bindIPC('goForward', function () {
     })
 })
 
+webviews.bindIPC('showBackArrow', function() {
+    var backArrow = document.getElementById('left')
+    backArrow.classList.toggle('shown')
+    setTimeout(function() {
+      backArrow.classList.toggle('shown')
+    }, 800)
```
Comment (PalmerAL) [inline js/webviews.js]: 800 seems a little too long, maybe somewhere around 500-600?

**Comment by PalmerAL** [comment]
I've added a few minor comments. Otherwise, this looks pretty good!

**Comment by LEoREo2247** [comment]
Alright, I've updated my code based on feedback.

```css
.arrow-indicator {
	width: 140px;
	height: 140px;
	border-radius: 70px;
	background: #000;
	position: fixed;
	top: 50%;
	z-index: 1;
	transition: opacity 0.3s, visibility 0.3s;
	opacity: 0;
	visibility: hidden;
}
```
Though, `visibility` seems to be needed. Otherwise, arrow indicator's fade-out animation won't be applied.

**Pull request merged**